src/screens/FeedScreen/Content.js

```diff
@@ -336,6 +331,12 @@ export const styles = StyleSheet.create({
     justifyContent: 'space-between',
     marginLeft: 13
   },
+
+  containerFeedText: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: 5
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new style `containerFeedText` has been added. As long as this new style is being used appropriately in the component and aligns with the overall design, this change is good.
```diff
+  containerFeedText: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: 5
+  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->